samples/dotnet/kernel-syntax-examples/Example17_ChatGPT.cs

```diff
@@ -10,71 +11,183 @@
 /**
  * The following example shows how to use Semantic Kernel with OpenAI ChatGPT API
  */
-
 // ReSharper disable once InconsistentNaming
-// ReSharper disable CommentTypo
 public static class Example17_ChatGPT
```

Comment:
in the spirit of having basic syntax examples I would move all the streaming examples in a separate example dedicated to streaming

Reply:
@dluc 
When I'm a end user the first thing I would look is for ChatGPT example and then I would see how many different ways I could do it. That's why I put the streaming in here... moving this to streaming in the end of the line will mostly hide it and enforce the End user to know that there is this `StreamingChat` that do the same but 20 examples further... 

I believe we can actually improve how we present our samples, is getting big.

Reply:
Yeah, I'd rather condense the syntax examples into logical domains, based on likely user lookups. They're becoming too numerous and, with no particular order or clustering, it's becoming difficult to find what you need.